runtime/dancebox/src/lib.rs

```diff
@@ -704,13 +724,65 @@ impl pallet_authority_assignment::Config for Runtime {
     type AuthorityId = NimbusId;
 }
 
+pub const FIXED_BLOCK_PRODUCTION_COST: u128 = 1 * currency::MICRODANCE;
+
+pub struct BlockProductionCost<Runtime>(PhantomData<Runtime>);
+impl ProvideBlockProductionCost<Runtime> for BlockProductionCost<Runtime> {
+    fn block_cost(_para_id: &ParaId) -> (u128, Weight) {
+        (FIXED_BLOCK_PRODUCTION_COST, Weight::zero())
+    }
+}
+
+parameter_types! {
+    // 60 days worth of blocks
+    pub const MaxCreditsStored: BlockNumber = 60 * 7200;
```

Comment:
can we make this a constant? (the 7200 I mean), something like what we have in constants BLOCKS_PER_DAY

Reply:
It already existed, its called DAYS